_pages/welcome-to-TTS/classes/olu.md

```diff
@@ -24,6 +24,13 @@ Required classes will be assigned in OLU and pushed individually to the OLU home
 * Plain Language
 * What is Record Management?
 
+[Here](Insite.gsa.gov/mandatorytraining) is a full list of all GSA mandatory employee and supervisor trainings
```

Comment:
Using "here" as a link signifier is not super friendly for screen readers. How about this instead:

`[You can see a full list of all GSA mandatory employee and supervisor trainings on Insite.](https://insite.gsa.gov/employee-resources/training-and-development/mandatory-training)`